README.md

```diff
@@ -60,7 +61,7 @@ Visit [db_server](https://github.com/gautamkrishnar/nothing-private/tree/master/
 #### Technologies used
 
 * [Client.js Browser fingerprinting](https://github.com/jackspirou/clientjs)
-* [PHP](https://secure.php.net/)
+* [PHP](https://www.php.net/)
```

Comment:
I don't think that this change is required anymore. www subdomain will work now. Can you please revert the change: https://github.com/docker-library/php/pull/809